src/contribute/ptransform-style-guide.md

```diff
@@ -172,7 +172,8 @@ Data processing is tricky, full of corner cases, and difficult to debug, because
 Do:
 
 * Generally, follow the rules of [semantic versioning](http://semver.org/).
-* If the API of the transform is not yet stable, mark it `@Experimental`.
+* If the API of the transform is not yet stable, annotate it as `@Experimental` (Java) or `@experimental` ([python](https://beam.apache.org/documentation/sdks/pydoc/2.0.0/apache_beam.utils.html#module-apache_beam.utils.annotations)).
```

Comment:
For these two links, you can use {{ site.release_latest }} instead of 2.0.0 so it will stay up to date with the latest release. also for consistency, can you capitalize the two references to "python" as the link text?